src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/FormTheme/specific_prices.html.twig

```diff
@@ -34,3 +34,14 @@
 {% block specific_prices_list %}
   {% include '@PrestaShop/Admin/Sell/Catalog/Product/SpecificPrice/Blocks/list.html.twig' %}
 {% endblock %}
+
```

Comment:
One issue I can't resolve, this part works if I put it in prestashop_ui_kit.html.twig but won't work if I just leave it here. Specific prices part works fine so I am not sure what's wrong.